src/test/groovy/au/org/ala/specieslist/SecurityInterceptorSpec.groovy

```diff
@@ -117,7 +117,7 @@ class SecurityInterceptorSpec extends Specification {
 
     public static class MockQueryService extends QueryService {
         @Override
-        def constructWithFacets(String base, List facets, String dataResourceUid) {
+        def constructWithFacets(String base, List facets, String dataResourceUid, String q) {
```

Comment:
If you made this:
```groovy
def constructWithFacets(String base, List facets, String dataResourceUid, String q = "") {
```

then you don't need to change all the method calls above, where you added `..., null)` at the end. Groovy will still match this method with only 3 args...